devel/libwebsockets/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; indent-tabs-mode: nil -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        warmcat libwebsockets 2.4-stable v
```

Comment:
Add a version line so it is "2.4" instead of "2.4-stable" for the Portfile.
`version      2.4`